docs/coordinates/spectralcoord.rst

```diff
@@ -5,13 +5,9 @@ Using the SpectralCoord Class
 
 .. warning::
 
-    The |SpectralCoord| class is new in Astropy v4.1 and should be considered
```

Comment:
This one perhaps we should keep, as 5.0 will be the first LTS with `SpectralCoord`. That said, I am happy with the cleanup.

Reply:
I'll restore the previous version then.